codes/c/chapter_backtracking/preorder_traversal_i_compact.c

```diff
@@ -0,0 +1,85 @@
+/**
+ * File: preorder_traversal_i_compact.c
+ * Created Time: 2023-05-10
+ * Author: Gonglja (glj0@outlook.com)
+ */
+
+#include "../utils/common.h"
+
+#define MaxLength 1024
+
+/* TreeNode 动态数组简单实现 */
+struct myArray {
```

Comment:
Can we replace myArray with the public module `vector`?

Reply:
Updated.

Reply:
> Can we replace myArray with the public module `vector`?

Updated.